slack/translations/strings.js

```diff
@@ -0,0 +1,6 @@
+export default messages = {
+    "string1": "Hello",
```

Comment:
This object is missing the comments mentioned in line 8 of the `README.md`. Should we add them to show how the generated XLIFF file will look in its entirety?

Reply:
comments added